### PR TITLE
feat(meeting): add claiming a PMR functionality

### PR DESCRIPTION
**Pull request by chrisadubois**

### CODE REVIEW

File: packages/node_modules/@webex/plugin-meetings/src/constants.js [hunk 1]
```diff
@@ -66,6 +70,9 @@ export const TYPE = 'type';
 
 export const USE_URI_LOOKUP_FALSE = 'useUriLookup=false';
 
+export const VALID_PMR_ADDRESS = /([a-z0-9][-a-z0-9, '.']{0,62})@([a-z0-9][-a-z0-9, '.']{0,62})\.webex\.com/i;
```
Comment (mattnorris) [inline packages/node_modules/@webex/plugin-meetings/src/constants.js]: Do you have a regex reference for these like in your other PR? I liked that you included that.

File: packages/node_modules/@webex/plugin-meetings/src/personal-meeting-room/request.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+  /**
+   *
+   * @param {Object} options with format of {userId, passcode, meetingAddress, preferred}
+   * @returns {Promise} returns a promise that resolves/rejects the result of the request
+   * @memberof PersonalMeetingRoomRequest
+   */
+  claimPmr(options) {
+    if (!options || !options.userId || !options.passcode || !options.meetingAddress || !options.preferred) {
+      throw new Error('Claiming a PMR should be done with userId, passcode, meetingAddress, and pereferred fields.');
```
Comment (mattnorris) [inline packages/node_modules/@webex/plugin-meetings/src/personal-meeting-room/request.js]: "and preferred fields."

File: packages/node_modules/@webex/plugin-meetings/src/personal-meeting-room/request.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    }
+    const validPin = PersonalMeetingRoomUtil.getClaimPmrPin(options.passcode);
+    if (!validPin) {
+      return Promise.reject(new Error('The host pin provided was of an invalid format'));
+    }
+    const validLink = PersonalMeetingRoomUtil.getClaimPmrLink(options.meetingAddress);
+    if (!validLink) {
+      return Promise.reject(new Error('The pmr link provided was of an invalid format'));
```
Comment (mattnorris) [inline packages/node_modules/@webex/plugin-meetings/src/personal-meeting-room/request.js]: Capitalize the acronym to "PMR" for logs, messages, etc.

File: packages/node_modules/@webex/plugin-meetings/src/personal-meeting-room/request.js (hunk 2 above)
Comment (chrisadubois) [reply packages/node_modules/@webex/plugin-meetings/src/personal-meeting-room/request.js]: will update the string

File: packages/node_modules/@webex/plugin-meetings/src/constants.js (hunk 1 above)
Comment (chrisadubois) [reply packages/node_modules/@webex/plugin-meetings/src/constants.js]: I don't for all of them, some of them just coming from web client ;/

File: packages/node_modules/@webex/plugin-meetings/src/personal-meeting-room/request.js (hunk 3 above)
Comment (chrisadubois) [reply packages/node_modules/@webex/plugin-meetings/src/personal-meeting-room/request.js]: Will do.

**Pull request merged**